From 9c35ae946acc77724efb7a19a2d19dd47f02daab Mon Sep 17 00:00:00 2001
From: Guillaume <guillaume.thouvenin@vates.tech>
Date: Thu, 26 Jun 2025 16:22:42 +0200
Subject: [PATCH] Bumping database schema version

A new field supported_image_format has been added to SM object and a new field
has been added to:
  - VM.migrate_send
  - VM.assert_can_migrate
  - VDI.pool_migrate

Signed-off-by: Guillaume <guillaume.thouvenin@vates.tech>
---
 ocaml/idl/datamodel_common.ml | 2 +-
 ocaml/idl/schematest.ml       | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/ocaml/idl/datamodel_common.ml b/ocaml/idl/datamodel_common.ml
index 50bc585b7..a044c9a0f 100644
--- a/ocaml/idl/datamodel_common.ml
+++ b/ocaml/idl/datamodel_common.ml
@@ -10,7 +10,7 @@ open Datamodel_roles
               to leave a gap for potential hotfixes needing to increment the schema version.*)
 let schema_major_vsn = 5
 
-let schema_minor_vsn = 786
+let schema_minor_vsn = 787
 
 (* Historical schema versions just in case this is useful later *)
 let rio_schema_major_vsn = 5
diff --git a/ocaml/idl/schematest.ml b/ocaml/idl/schematest.ml
index dcb0722a8..e6c48ea1a 100644
--- a/ocaml/idl/schematest.ml
+++ b/ocaml/idl/schematest.ml
@@ -3,7 +3,7 @@ let hash x = Digest.string x |> Digest.to_hex
 (* BEWARE: if this changes, check that schema has been bumped accordingly in
    ocaml/idl/datamodel_common.ml, usually schema_minor_vsn *)
 
-let last_known_schema_hash = "6f6230f87a92572b68ebd742196ffd0e"
+let last_known_schema_hash = "199b240031e758ce6014b5b47457ca2c"
 
 let current_schema_hash : string =
   let open Datamodel_types in
